package.json

```
@@ -74,6 +114,7 @@
     "tslib": "2.5.0",
     "typescript": "4.7.4",
     "websocket": "1.0.34",
+    "wireit": "^0.9.5",
```

Usually dependencies need to be pinned.
```suggestion
    "wireit": "0.9.5",
```